### PR TITLE
Instructions on how to create a cronjob for renewing Lets Encrypt certificates

### PR DESCRIPTION
**Pull request by danishkhan**
Lets Encrypt certificates need to be renewed every 90 days rather than a user forgetting to do that this provides them with instructions on how to create a cronjob to do that automatically. The only process they need to do externally is communicating with their users whenever there will be downtime for the renewal to take place.

### CODE REVIEW

File: user/enterprise/platform-tips.md [hunk 1]
```diff
@@ -253,7 +253,52 @@ $ sudo certbot renew
 $ replicatedctl app start
 ```
 
-In general: These certificate renewals should be automated with a cron job.
+Automate renewal with a cron job
```
Comment (philpeble) [inline user/enterprise/platform-tips.md]: Should this be a header of some sort?

**Comment by danishkhan** [comment]
Made the changes @ggcampinho-travis and @philpeble let me know if I wrote the line for the `chmod +x` in a way that makes sense for our users. If so, then I'll merge this in.

**Comment by ggcampinho-travis** [comment]
It's good for me 💛

File: user/enterprise/platform-tips.md [hunk 2]
```diff
@@ -253,7 +253,54 @@ $ sudo certbot renew
...
+}
+
+if [[ $# -ne 0 ]]; then
+  usage
+fi
+
+replicatedctl app stop
+sudo certbot renew
+replicatedctl app start
+```
```
Comment (plaindocs) [inline user/enterprise/platform-tips.md]:
```suggestion
```{: data-file="./home/ubuntu/renew-certs.sh"}
```

File: user/enterprise/platform-tips.md [hunk 3]
```diff
@@ -253,7 +253,54 @@ $ sudo certbot renew
 $ replicatedctl app start
 ```
 
-In general: These certificate renewals should be automated with a cron job.
+### Automate renewal of your Let's Encrypt SSL Certificate with a cron job
+
+Create a file such as `/home/ubuntu/renew-certs.sh` with the following in it:
+
+```
```
Comment (plaindocs) [inline user/enterprise/platform-tips.md]:
```suggestion
```sh
```

File: user/enterprise/platform-tips.md [hunk 4]
```diff
@@ -253,7 +253,54 @@ $ sudo certbot renew
...
+  usage
+fi
+
+replicatedctl app stop
+sudo certbot renew
+replicatedctl app start
+```
+Set the executable permissions you'll need by doing `chmod +x /home/ubuntu/renew-certs.sh`.
+
+You'll then want to create a cronjob by using your favorite text editor, for example:
```
Comment (plaindocs) [inline user/enterprise/platform-tips.md]: nano is like nobodies favourite editor! :-p How about 

> Edit `/etc/crontab` by appending the following lines:

File: user/enterprise/platform-tips.md [hunk 5]
```diff
@@ -253,7 +253,54 @@ $ sudo certbot renew
...
+
+```
+nano /etc/crontab
+```
+
+Then append the below to the file that you've just opened in your text editor. Make sure to adjust the configuration with values that apply to the certificate you are creating.
```
Comment (plaindocs) [inline user/enterprise/platform-tips.md]: How about adding a specific example instead the file comment.

File: user/enterprise/platform-tips.md [hunk 6]
```diff
@@ -253,7 +253,54 @@ $ sudo certbot renew
...
+
+```
+# Renews certs at 2am on the 1st of February, May, August, and November.
+# Please change the configuration that applies to the certificate you are creating.
+0 2 1 2,5,8,11 * /home/ubuntu/renew-certs.sh
+```
+
+**Note: Be aware that this process will also introduce downtime.**
```
Comment (plaindocs) [inline user/enterprise/platform-tips.md]: You can use quote `>` to add note formatting.

```suggestion
> This process will introduce a small amount of downtime while the certificates are renewed.
```

File: user/enterprise/platform-tips.md [hunk 7]
```diff
@@ -253,7 +253,54 @@ $ sudo certbot renew
...
+
+Make sure to plan to communicate with your users every time this downtime will happen so they are aware that their builds will temporarily be stopped until the certificate gets renewed and your Travis CI Enterprise installation comes back online.
```
Comment (plaindocs) [inline user/enterprise/platform-tips.md]:
```suggestion
Make sure to communicate with your users before each renewal so they are aware that their builds will temporarily be stopped until the certificate gets renewed.
```

**Pull request merged**